recipes/ctags/bld.bat

```diff
@@ -0,0 +1,4 @@
+nmake -f mk_mvc.mak
```

Comment:
Since this is compiling c source files, you need to compile it for different versions of `vc`. See http://conda-forge.github.io/docs/buildwin.html#vc-features. Let me know if you need help